doc/api/deprecations.md

```diff
@@ -2594,6 +2610,7 @@ Passing a callback to [`worker.terminate()`][] is deprecated. Use the returned
 [`util`]: util.html
 [`worker.exitedAfterDisconnect`]: cluster.html#cluster_worker_exitedafterdisconnect
 [`worker.terminate()`]: worker_threads.html#worker_threads_worker_terminate
+[`WriteStream.open()`]: fs.html#fs_class_fs_writestream
```

Comment:
This needs to be placed after the `` [`SlowBuffer`] `` (ASCII sorting is case-sensitive).